wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -264,6 +264,14 @@ public protocol UserSession: AnyObject {
 
     func makeSetConversationGuestsAndServicesUseCase() -> SetAllowGuestAndServicesUseCaseProtocol
 
+    func makeAppendTextMessageUseCase() -> any AppendTextMessageUseCaseProtocol
+
+    func makeAppendImageMessageUseCase() -> AppendImageMessageUseCaseProtocol
+
+    func makeAppendKnockMessageUseCase() -> any AppendKnockMessageUseCaseProtocol
+
+    func makeAppendLocationMessageUseCase() -> AppendLocationMessagekUseCaseProtocol
```

Comment:
Why are some `any` and the others are not?

Reply:
I don't we need any in this case. In the first iteration we were using an associated type now we don't. Fixed it here: https://github.com/wireapp/wire-ios/commit/f16a68f8b2a651cda63a05d18394980f82791160